core/server/middleware/check-ssl.js

```diff
@@ -44,7 +44,7 @@ checkSSL = function checkSSL(req, res, next) {
                 forceAdminSSL: config.get('forceAdminSSL'),
                 configUrlSSL: config.get('urlSSL'),
                 configUrl: config.get('url'),
-                reqUrl: req.url
+                reqUrl: req.originalUrl || req.url
```

Comment:
i saw this change a couple of times in the past. seems like this can be forgotten very easily. 
any idea how to prevent?